.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -223,23 +232,35 @@ jobs:
       uses: ./.github/actions/download-artifact-with-retry
       with:
         name: nvd
-        path: nvd.zip
+        path: .
```

Comment:
This is redundant since by default `download-artifact` will download to `GITHUB_WORKSPACE`, which is `.` AFAIK. Is there a reason why you selected `.`?

Also, can you confirm why the previous `path` value was wrong?

Reply:
My understanding is that `path: nvd.zip` means the output folder name is `nvd.zip`

Reply:
So, `.` is redundant and not needed.

Reply:
So we don't need the `path` config?

Reply:
I think we don't?

![image](https://github.com/user-attachments/assets/ec5586c5-874c-44d3-9d49-a18e0bc2810d)

Although, does not hurt to have it? You can decide. I will resolve.